common/retval.h

```diff
@@ -71,7 +71,7 @@ struct retval : public retval_base {
 
 template<>
 struct retval<void> : public retval_base {
-    retval(int _errno = 0) : retval_base{(uint64_t)_errno} { }
+    retval(int errno_ = 0) : retval_base{(uint64_t)errno_} { }
```

Comment:
photon/common/retval.h:74:16: warning: declaration of ‘_errno’ shadows a member of ‘photon::retval<void>’ [-Wshadow]